app-examples/src/main/protobuf/rest_routes_v2.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto2";
+
+option java_multiple_files = true;
+// Should be included in the service packages scanned by ServiceScan
+option java_package = "net.spals.appbuilder.app.examples.grpc.rest";
+option java_outer_classname = "RestServerV2Proto";
+
+import "google/api/annotations.proto";
+
+service UserServiceV2 {
+    rpc DeleteUserV2(DeleteUserRequestV2) returns (DeleteUserResponseV2) {
+        option (google.api.http).delete = "/v2/users/{id}";
```

Comment:
In order to enable REST+JSON for a gRPC server, you need to annotate the `rpc` statements with a REST endpoint.